src/pipeline-cli/is-openshift-list.ts

```diff
@@ -0,0 +1,5 @@
+import isPlainObject from 'lodash.isplainobject'
+
+export function isPlainObj(object: any): boolean {
```

Comment:
Shoud be: isOpenshiftList?

Reply:
@mbystedt Yes, but I was told not to take any liberties bringing in pipeline-cli and nr-pipeline-ext.  That would be a definite fix on the next pass.  Thanks for reviewing!  :)

Original:
https://github.com/BCDevOps/pipeline-cli/blob/master/lib/isOpenShiftList.js